tests/sentry/web/frontend/test_error_500.py

```diff
@@ -2,9 +2,11 @@
 from django.urls import reverse
 
 from sentry.testutils import TestCase
+from sentry.testutils.silo import control_silo_test
 
 
 @override_settings(ROOT_URLCONF="sentry.conf.urls")
+@control_silo_test(stable=True)
```

Comment:
This actually importaly should be an all_silo_test so we validate this logic is silo independent.